lib/plugins/helpers/docker.rb

```diff
@@ -48,9 +48,29 @@ def manage_set
   end
 
   def build
+    if Docker::Image.exist?(@name) && !@config['plugin']['build_force']
+      built_image
+    else
+      build_image
+    end
+  end
+
+  def built_image
```

Comment:
Skimming this the naming is very close.  Possibly a comment or a method header comment may help.

Otherwise, looks very clean 👍 from me.